configurations/pipeline_config.json

```diff
@@ -133,6 +133,7 @@
     {"RapidProKey": "Young_People (Value) - mag_s01_w10_follow_up", "PipelineKey": "young_people_issues_raw"},
     {"RapidProKey": "Young_People (Run ID) - mag_s01_w10_follow_up", "PipelineKey": "young_people_issues_run_id"}
   ],
+  "PipelineName": "main_pipeline",
```

Comment:
If this is for the 'main_pipeline', or 'all_quarters', why don't we name this file as such

Reply:
Updated thanks!